tutorials/index.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Tutorials
+suppressRelated: true
+---
+
+These tutorials will give you step-by-step instruction to help you learn NServiceBus.
+
+### [NServiceBus 101: Messaging Basics](nservicebus-101/)
+
+In this course, you will learn how to get working quickly with NServiceBus, including how to set up your development environment and create messaging endpoints, defining messages and handlers, how to send messages and publish events, and how to deal with exceptions and take advantage of automatic retries.
+
+### Additional courses
+
+Please [sign up for updates](https://particular.net/s/new-course-signup) if you'd like to be notified when we release additional courses.
```

Comment:
Isn't that just a CTA to gauge interest? If so, can we keep it temporarily and revisit when results will be evaluated?